src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -12,9 +12,31 @@ import {
   evidenceTypesDescription,
   EvidenceTypeHelp,
   disabilityNameTitle,
+  facilityDescription,
+  treatmentView,
   vaMedicalRecordsIntro
 } from '../helpers';
 
+const {
+  treatments
+} = fullSchema526EZ.properties;
+
+const {
+  date
+} = fullSchema526EZ.definitions;
+
+// We may add these back in after the typeahead, but for now...
+const treatmentsSchema = _.cloneDeep(treatments);
```

Comment:
This seems unnecessary. You could skip and use _.set instead of the assignment statement below.

Reply:
Good point. I forgot you can set deeply nested properties like this. :man_facepalming: